sdk/python/kubeflow/training/api/training_client.py

```diff
@@ -81,6 +89,109 @@ def __init__(
             )
         self.job_kind = job_kind
 
+    def train(
+        self,
+        name: str = None,
+        namespace: str = None,
+        num_workers: int = 1,
+        num_procs_per_worker: int = 1,
+        pvc: Dict[Literal["name", "claimName"], str] = None,
+        model_params: HuggingFaceModelParams = None,
+        dataset_params: S3DatasetParams = None,
+        parameters: HuggingFaceTrainParams = None,
```

Comment:
@johnugeorge @deepanker13 Should we call it `train_parameters`, similar to `dataset_parameters` ?
I guess, in this API, parameters are not the same as parameters that we set for `create_job` API.